### ISSUE TITLE
Typo in clause for predicate clauselength/3

### ISSUE THREAD

**Issue by pmoura** [vscosta/yap-6.3#42]
Typo in clause for predicate `clauselength/3`(found by the Logtalk linter):

https://github.com/vscosta/yap-6.3/blob/13d88949c95ced7a84999a950ee37d4723e8a278/library/clauses.yap#L86-L89

It should be:

```logtalk
clauselength( C, I1, I ) :- 
    var(C), 
    !, 
    I1 is I+1.
```

**Comment by vscosta** [comment]
Makes sense, thanks!

**Comment by pmoura** [comment]
Fixed as of version YAP 6.5.0-6629c13c.
